MSProject/ViewControllers/DetailBook/WBDetailBookViewController.swift

```diff
@@ -54,13 +50,27 @@ class WBDetailBookViewController: UIViewController {
 
         viewModel.rentBookAction.values.observeValues { [unowned self] _ in
             MBProgressHUD.hide(for: self._view, animated: true)
-            self.showAlertMessage(message: "BOOK_RENTED".localized())
+            self.showAlert(message: "BOOK_RENTED".localized())
+            
+            // trampita para actualizar la vista segun el book view model
+            if let bookViewModel = self.bookViewModel {
```

Comment:
I don't understand what is this doing 🤔

Reply:
cause I need to change the book status from available to rented, the backend doesnt bring that information untill call the /books service. A new way to do this is coming... I will handle the books locally to match with the rented books and wishes books. We can discuss it for a better way to do this! =)